### DISCUSSION TITLE
Add rename layout in `View Manager`

### DISCUSSION THREAD

**Discussion by brunodavi** [Sketchware-Pro/Sketchware-Pro#473] [Ideas]
Good afternoon, I would like you to develop a way to rename the activity views or custom views

<details>
<summary>Example</summary>

![view_manager](https://user-images.githubusercontent.com/48166556/152648057-2d30cd41-003f-4232-9190-07cc6c437284.jpg)

> In this case this happened to me I created the custom one but I need to rename it to keyboard view and there is only the option to remove

</details>

**Comment by brunodavi** [comment]
😅 Could not create `proposal` label in github app

**Comment by j-px** [comment]
It's honestly probably pretty unlikely to be implemented. Blocks might be depending on that Activity / Custom View, and I'm not sure if there can be code to replace all references.

Another "large?" precondition: the activity must either get decompiled & get reimplemented in Java, or modified on low level with Smali.

**Comment by brunodavi** [comment]
I understand, thanks for letting me know
